go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
 	github.com/go-logr/logr v1.2.0
 	github.com/identitatem/dex-operator v0.0.4-0.20220315143428-0926431039aa
-	github.com/identitatem/idp-client-api v0.0.0-20220223205659-c83d29bc377e
+	github.com/identitatem/idp-client-api v0.0.0-20220304183352-0981ca52c04b
```

Comment:
Might need to change this as it looks old now and would backlevel idp-client-api

Reply:
This PR works with that client-api, the one you just lgtm will be used in a PR that I didn't created yet.